acceptance-tests/src/test/java/org/eclipse/collections/impl/forkjoin/FJIterateAcceptanceTest.java

```diff
@@ -291,7 +291,7 @@ public void testForEachWithException()
     public void testForEachWithIndexToArrayUsingFastListSerialPath()
     {
         Integer[] array = new Integer[200];
-        FastList<Integer> list = (FastList<Integer>) Interval.oneTo(200).toList();
+        MutableList<Integer> list = Interval.oneTo(200).toList();
```

Comment:
I would write the right side of the equals as `new FastList<>(Interval.oneTo(200))` so it is clear this is still a `FastList` as the method name suggests. I know `toList()` will return a `FastList`, but I think it is better to be explicit.